ubuntu_requirement_installation.sh

```diff
@@ -0,0 +1,126 @@
+#!/usr/bin/env bash
+set -ex
+
+source lib/logging.sh
+
+# sudo apt install -y libselinux-utils
+# if selinuxenabled ; then
+#     sudo setenforce permissive
+#     sudo sed -i "s/=enforcing/=permissive/g" /etc/selinux/config
+# fi
+
+# Update to latest packages first
+sudo apt -y update
+
+# Install EPEL required by some packages
+# if [ ! -f /etc/yum.repos.d/epel.repo ] ; then
+#     if grep -q "Red Hat Enterprise Linux" /etc/redhat-release ; then
+#         sudo yum -y install http://mirror.centos.org/centos/7/extras/x86_64/Packages/epel-release-7-11.noarch.rpm
+#     else
+#         sudo yum -y install epel-release --enablerepo=extras
+#     fi
+# fi
+
+# Work around a conflict with a newer zeromq from epel
+# if ! grep -q zeromq /etc/yum.repos.d/epel.repo; then
+#   sudo sed -i '/enabled=1/a exclude=zeromq*' /etc/yum.repos.d/epel.repo
+# fi
+
+# Install required packages
+# python-{requests,setuptools} required for tripleo-repos install
+sudo apt -y install \
+  net-tools \
+  crudini \
+  curl \
+  dnsmasq \
+  figlet \
+  golang \
+  nmap \
+  patch \
+  psmisc \
+  python-pip \
```

Comment:
All the python-* should be installed in a virtual env like ansible

Reply:
I'm also considering that should we use apt or pip to install Python packages. I consulted the ideas in this page: https://unix.stackexchange.com/questions/450008/pip-vs-package-manager-for-handling-python-packages. 
What they said make sense.
How do you think about this problem ?

Reply:
If we use a virtual environment, we won't be impacted by or impacting the python distribution installed, we'll be completely independent, so we shouldn't hit such problems

Reply:
Ok. So why should we install all python-* in a virtual env ?